src/Compilers/Core/Portable/SpecialType.cs

```diff
@@ -263,9 +263,14 @@ public enum SpecialType : sbyte
         /// </summary>
         System_Runtime_CompilerServices_InlineArrayAttribute = 46,
 
+        /// <summary>
+        /// Indicates that the type is <see cref="ReadOnlySpan{T}"/>
```

Comment:
>Indicates that the type is <see cref="ReadOnlySpan{T}"/> [](http://example.com/codeflow?start=12&length=56)

We need to make it clear, that in a general scenario this item cannot be used to find the"canonical" definition of `ReadOnlySpan` type that is used by compiler.